frontend/packages/operator-lifecycle-manager/integration-tests-cypress/views/operator.view.ts

```diff
@@ -201,4 +208,5 @@ export type TestOperandProps = {
   kind: string;
   tabName: string;
   exampleName: string;
+  deleteUrl?: string;
```

Comment:
Console naming conventions would make this

```suggestion
  deleteURL?: string;
```